packages/blockchain-extension/test/commands/deleteEnvironmentCommand.test.ts

```diff
@@ -427,22 +419,20 @@ describe('DeleteEnvironmentCommand', () => {
 
         });
 
-        it(`should be able to delete a local environment from the tree`, async () => {
-            const settings: any = {};
-            settings['otherLocal'] = {
-                ports: {
-                    startPort: 17050,
-                    endPort: 17070
-                }
-            };
-            await vscode.workspace.getConfiguration().update(SettingConfigurations.FABRIC_RUNTIME, settings, vscode.ConfigurationTarget.Global);
+        it(`should be able to delete a local microfab from the tree`, async () => {
+
+            const originalSettings: any = vscode.workspace.getConfiguration().get(SettingConfigurations.FABRIC_RUNTIME, vscode.ConfigurationTarget.Global);
+            const newSettings: any  = originalSettings;
+            newSettings['otherLocal'] = 8081;
```

Comment:
This caught my attention because not knowing what the return type is I can't tell if this is a real copy or a copy by reference, which wouldn't be good.
Having ran this test locally, I can't see any difference between `originalSettings` and `newSettings`, even after setting ['otherLocal'] to a different port.

I also added `localSettings.should.deep.equal(newSettings);` at the end of the test and it still passes.

Not sure what to make of it, is this expected?

Reply:
Rubbish test, have now updated it - good spot!